pkg/collector/corechecks/servicediscovery/usm/jee.go

```diff
@@ -300,16 +300,16 @@ func (je jeeExtractor) doExtractContextRoots(extractor vendorExtractor, app *jee
 // extractServiceNamesForJEEServer takes args, cws and the fs (for testability reasons) and, after having determined the vendor,
 // If the vendor can be determined, it returns the context roots if found, otherwise the server name.
 // If the vendor is unknown, it returns a nil slice
```

Comment:
This comment needs updating with the added return value.

Reply:
Done.